### ISSUE TITLE
problem using rewrite in the module

### ISSUE THREAD

**Issue by FarshadHadei** [claranet/terraform-azurerm-app-gateway#2]
Hi!, I'm trying to set rewrite but getting the following error:

`Invalid value for "inputMap" parameter: lookup() requires a map as the first
argument.`

These are the values I used for my code:

```appgw_rewrite_rule_set = {
    rewrite_rule_set = {
      name = "loyal-to-loyal999"
      rewrite_rule = {
        name          = "loyal-to-loyal999"
        rule_sequence = "100"
        condition_ignore_case = true
        condition_negate      = false
        condition_pattern     = "loyal.xyz.net"
        condition_variable    = "http_req_Host"
        response_header_name  = "Host"
        response_header_value = "loyal999.xyz.net"
      }
    }
  }
```
Please advise and thanks!

**Comment by rossifumax** [comment]
Hello,
Thanks for reporting this.

We're already working on rewriting the `rewrite_rule_set` block, but actually we're facing an issue depending on the AzureRM provider, so we're blocked for the moment on this issue.

Please take a look at the issue we've opened on the AzureRM provider : https://github.com/terraform-providers/terraform-provider-azurerm/issues/11611

We'll keep you informed
Thanks

**Comment by Shr3ps** [comment]
hello,

this should be fixed by the release v4.8.0 (https://github.com/claranet/terraform-azurerm-app-gateway/blob/master/CHANGELOG.md#v480---2021-06-25)
